### PR TITLE
New script to search notes via title

### PR DESCRIPTION
**Pull request by commai**
## Description

Simply allows you to search your notes via title. 

Note: titles need to be exact matches in order to query them properly and in the event that you have mutliple of the same titles, it'll choose the most recently edited one to display. 

## Type of change
- [x] New script command

## Screenshot
https://user-images.githubusercontent.com/46573539/127367109-8e84d725-2178-40c4-a1c4-0d68c13f325b.mov

## Dependencies / Requirements

No reqs/depedencies required. 

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by RSO** [comment]
Thank you for the contribution! Do you know whether it'd be possible to show a more helpful error message in the case that the note can't be found? 

It's too bad that it requires the note title to be an exact match, and that there does not seem to be search functionality.

**Comment by unnamedd** [comment]
I had the same though as you @RSO! This Script Command may be called "Open Note" instead of "Search Notes". 

> Thank you for the contribution! Do you know whether it'd be possible to show a more helpful error message in the case that the note can't be found?
> 
> It's too bad that it requires the note title to be an exact match, and that there does not seem to be search functionality.

**Comment by commai** [comment]
Yup for sure! I'll go ahead and add a clear error message for a failed search, and will update the name to be open note as opposed to search note.

**Comment by commai** [comment]
@unnamedd @RSO 

I've gone ahead and added some better error logging, and updated the name to (Open Note as opposed to Search Note)

https://user-images.githubusercontent.com/46573539/127708757-7291440a-8379-4cee-9b96-84b6e06989e1.mov

File: commands/apps/notes/open-note.applescript [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# @raycast.description Open Note via its Title
+# @raycast.author Vardan Sawhney
+# @raycast.authorURL https://github.com/commai
+
+on run argv
+	tell application "Notes"
+		if exists note (item 1 of argv)
+			show note (item 1 of argv)
+		else 
+			log "Sorry, a note with that title was not found"
```
Comment (unnamedd) [inline commands/apps/notes/open-note.applescript:23]: I didn’t test, but what do you think?

```suggestion
			log "Sorry, note '" & (item 1 of argv) & "' was not found" 
```

Comment (commai) [reply commands/apps/notes/open-note.applescript]: Made an update to the script that's cleaner: `Sorry, the note "titleofnote" was not found` is an example of what the output would look like.

**Review by unnamedd — APPROVED** [review]
Nice! Thanks for the changes @commai!

**Review by fe9lix — APPROVED** [review]

**Pull request merged**